### PR TITLE
[WebNN EP] Support Gelu op

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@fdwr, @guschmue, PTAL, thanks!

**Review by guschmue — DISMISSED** [review]

File: onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc [hunk 1]
```diff
@@ -66,13 +72,35 @@ Status ActivationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
...
+                                            const logging::Logger& logger) const {
+  const auto& input_defs = node.InputDefs();
+
+  std::vector<int64_t> input_shape;
+  if (!GetShape(*input_defs[0], input_shape, logger))
+    return false;
+
+  if (node.OpType() == "Gelu") {
+    NodeAttrHelper helper(node);
+    if (helper.Get("approximate", "none") != "none") {
```
Comment (fdwr) [inline onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc:88]: We should just return true in this case. I would be more worried about the *reverse* case (if we supported approximate `tanh` but didn't support `erf`), but if `erf` is more accurate than `approximate`, then it yields no benefit to precision to fall back to the CPU, and it hurts perf badly to split partitions.

**Review by fdwr — APPROVED** [review]
👍

**Pull request merged**